master/test/streamdata/stream_data.go

```diff
@@ -224,17 +204,22 @@ func AddExperiment(pgDB *db.PgDB) (int, error) {
 		return 0, err
 	}
 
-	_, err = db.Bun().NewInsert().Model(
-		&experiment{
-			ID:                   ids[0] + 1,
-			JobID:                newJobID,
-			State:                "ERROR",
-			Notes:                "",
-			Config:               expconf.LegacyConfig{},
-			ModelDefinitionBytes: nil,
-			StartTime:            time.Now(),
-			OwnerID:              &ownerID,
-		}).Exec(ctx)
+	// we have to control id and jobId generation
+	experiment.ID = ids[0] + 1
+	experiment.JobID = newJobID
+
+	_, err = db.Bun().NewInsert().Model(experiment).Exec(ctx)
+	// example experiment:
+	// Experiment{
+	//			ID:                   ids[0] + 1,
```

Comment:
question (non-blocking): what are we gaining from this comment that's not being communicate in the declaration of this type?

Reply:
I added an XXX to the comment - it was purely for my benefit when I start writing experiment intg tests

Reply:
Makes sense!